Finance/001-options.1h.go

```diff
@@ -0,0 +1,83 @@
+//usr/bin/env go run $0 $@; exit
+//<xbar.title>Options Call Prices</xbar.title>
+//<xbar.version>1.0</xbar.version>
+//<xbar.author>Aaron Edell</xbar.author>
+//<xbar.author.github>aaronedell</xbar.author.github>
+//<xbar.desc>Track Specific Option Call Quotes</xbar.desc>
+//<xbar.image>https://d33wubrfki0l68.cloudfront.net/ea583c177bf610b4db04246ef394d583201feae7/870d6/assets/img/logo.png</xbar.image>
```

Comment:
Hey @aaronedell congrats on this plugin - can you provide an image of the plugin in use?

Reply:
I've committed the change to my branch. Do I have to do another pull request?